test/python/quantum_info/test_sparse_z2_symmetries.py

```diff
@@ -16,7 +16,7 @@
 
 from test.python.opflow import QiskitOpflowTestCase
```

Comment:
Given that we plan to deprecate opflow soon, I would not use their test cases. I would suggest:

```suggestion
from qiskit.test import QiskitTestCase
```

Reply:
(I cannot add the suggestion but you would also have to change line 23)

Reply:
Thanks, I had not realised. The change has been made